client/additional-methods-setup/wizard/wrapper/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React, { useState, useMemo } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import WizardContext from './context';
+
+const Wizard = ( {
+	children,
+	defaultActiveTask = '',
+	defaultCompletedTasks = {},
+} ) => {
+	const [ activeTask, setActiveTask ] = useState( defaultActiveTask );
+	const [ completedTasks, setCompletedTasks ] = useState(
+		defaultCompletedTasks
+	);
+
+	const contextValue = useMemo(
+		() => ( {
+			activeTask,
+			setActiveTask,
+			completedTasks,
+			setCompletedTasks,
+		} ),
+		[ activeTask, setActiveTask, completedTasks ]
+	);
+
+	return (
+		<WizardContext.Provider value={ contextValue }>
```

Comment:
Since this returns a Provider, could we call this file provider.js instead of index.js? That way it's clear that it's the Provider.

Reply:
Renamed `client/additional-methods-setup/wizard/wrapper/index.js` to `client/additional-methods-setup/wizard/wrapper/provider.js`